### PR TITLE
Fix Up Unused Laser Cannon Superweapon

### PR DESCRIPTION
**Pull request by commy2**
I admit, this is stupid.

- fixes the GUI:Lazr_LaserCannon text that appears in place of the Laser Cannon super weapon timer
- fixes various other text references to the "Particle Cannon" as well as the timer of "4 minutes" being wrong (it's 2)
- replaces all blue effects by red effects to distinguish it from the vanilla Particle Cannon
- makes the Laser Cannon reveal itself like other super weapons.

### CODE REVIEW

**Comment by xezon** [comment]
What is this Laser Cannon? I assume an optional entity not part of main factions?

**Comment by commy2** [comment]
Carbon copy of the Particle Cannon for Lasergen with some altered stats ($2000 instead of $5000, requires Airfield instead of Strat Center, longer time the beam is alive/different stats on how much and frequently it does damage, 2 min recharge time), but the same visuals.

File: Patch104pZH/GameFilesEdited/Data/INI/Object/LaserGeneral.ini [hunk 1]
```diff
@@ -19043,8 +19046,126 @@ Object Lazr_AmericaLaserCannon
...
+  ; *** ENGINEERING Parameters ***
+  KindOf = IMMOBILE
+
+  ClientUpdate = LaserUpdate ModuleTag_02
+    ;MuzzleParticleSystem = None
+    ;TargetParticleSystem = None
+  End
+End
+
+; Patch104p @feature commy2 23/08/2022 Beams emerging from SWG Particle Cannon when about to blow up. Uses pink texture instead of blue.
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/LaserGeneral.ini]: This has SWG comment.

**Review by xezon — APPROVED** [review]

**Comment by xezon** [comment]
@commy2 
When you "Merge with Rebase" instead of "Merge and Squash", then you must give proper description to each commit. The way this was submitted now is as "comments" and "Laser Cannon"

**Comment by commy2** [comment]
Oof, sorry.

**Comment by xezon** [comment]
No problem. I can try fix it later, merge them and rename and push main. Just don't merge anything else meanwhile.

**Comment by commy2** [comment]
Rebase master \o/

**Comment by xezon** [comment]
YOLO

**Comment by MTKing4** [comment]
Works.
this Looks Awesome 😄 

image

tooltip is more broken now than ever, GUI name Unchanged

image

**Comment by commy2** [comment]
Hmm, strange that none of the text fixes apply to you. Do you use custom hotkeys? If so, you don't get any of the text changes of the patch, because your generals.csf/.str does contain only the old texts.

**Comment by MTKing4** [comment]
Ah, that's probably it then

**Pull request merged**